crates/tako/src/internal/worker/rqueue.rs

```diff
@@ -15,14 +15,30 @@ type QueuePriorityTuple = (Priority, Priority, Priority); // user priority, reso
 pub(crate) struct QueueForRequest {
     resource_priority: Priority,
     queue: PriorityQueue<TaskId, PriorityTuple>,
+    is_blocked: bool,
 }
 
 impl QueueForRequest {
+    pub fn init(&mut self) {
```

Comment:
```suggestion
    pub fn reset_blocked(&mut self) {
```
Or something like that? `init` is a bit opaque.

Reply:
I originally named it "reset" but I want to make it compatible with other "init" functions. But you are right. I will rename also other inits to reset.